glean-core/src/ping/mod.rs

```diff
@@ -214,7 +214,7 @@ impl PingMaker {
     /// The directory will be created inside the `data_path`.
     /// The `pings` directory (and its parents) is created if it does not exist.
     fn get_pings_dir(&self, data_path: &Path) -> std::io::Result<PathBuf> {
-        let pings_dir = data_path.join("pings");
+        let pings_dir = data_path.join("pending_pings");
```

Comment:
I would be nice to have a way to share constants between Rust and Kotlin/Swift etc.  Maybe there's a cbindgen config for that?  Anyway, not necessary here at all...

Reply:
I [just filed a bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1582448) about this!